src/query/sql/src/planner/binder/udf.rs

```diff
@@ -133,22 +133,15 @@ impl Binder {
                 }
                 let return_type = DataType::from(&resolve_type_name(return_type, true)?);
 
-                if !["python", "javascript"].contains(&language.to_lowercase().as_str()) {
+                if !["python", "javascript", "wasm"].contains(&language.to_lowercase().as_str()) {
                     return Err(ErrorCode::InvalidArgument(format!(
                         "Unallowed UDF language '{language}', must be python or javascript"
```

Comment:
`must be python, javascript or wasm

Reply:
You're correct. Thank you for pointing that out. I have made the necessary correction.